.github/workflows/ci.yml

```diff
@@ -0,0 +1,132 @@
+name: Alfresco Transform Core CI
+
+on:
+  push:
+    branches:
+      - feature/**
+      - fix/**
+      - SP/**
+      - HF/**
+      - ATS-**
+      - ACS-**
+      - MNT-**
+  pull_request:
+    branches:
+      - master
+      - feature/**
+      - fix/**
+      - SP/**
+      - HF/**
+      - ATS-**
+      - ACS-**
+      - MNT-**
+  workflow_call:
+  workflow_dispatch:
+
+env:
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  JAVA_VERSION: "17"
```

Comment:
Is it necessary? setup-java-build from alfresco-build-tools already has Java 17 by default.